docs/hazmat/primitives/asymmetric/index.rst

```diff
@@ -32,4 +32,36 @@ and Elliptic Curve.
     serialization
     utils
 
+.. module:: cryptography.hazmat.primitives.asymmetric
+
```

Comment:
Sounds good.

On Mon, Feb 16, 2015 at 6:19 PM, Paul Kehrer <notifications@github.com>
wrote:

> In docs/hazmat/primitives/asymmetric/index.rst
> <https://github.com/pyca/cryptography/pull/1666#discussion_r24782353>:
>
> > @@ -32,4 +32,36 @@ and Elliptic Curve.
> >      serialization
> >      utils
> >
> > +.. module:: cryptography.hazmat.primitives.asymmetric
> > +
>
> We could create a docs/hazmat/primitives/asymmetric/interfaces.rst and
> link it in the toc but define its module as
> cryptography.hazmat.primitives.asymmetric so the module path links work
> properly.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/1666/files#r24782353>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084